sound/soc/intel/common/soc-acpi-intel-ptl-match.c

```diff
@@ -710,6 +733,13 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_ptl_sdw_machines[] = {
 		.sof_tplg_filename = "sof-ptl-rt712-l3-rt1320-l3.tplg",
 		.get_function_tplg_files = sof_sdw_get_tplg_files,
 	},
+	{
+		.link_mask = BIT(3),
+		.links = ptl_sdw_rt721_l3_rt1320_l3,
+		.drv_name = "sof_sdw",
+		.sof_tplg_filename = "sof-ptl-rt721-l3-rt1320-l3.tplg",
+		.get_function_tplg_files = sof_sdw_get_tplg_files,
+	},
```

Comment:
Can we use https://github.com/thesofproject/linux/pull/5596 instead of adding matches for PTL?

Reply:
> Thanks for sharing. I would test it. Before that I assume this requires SDCA Disco table support, right? Currently, Disco SDCA functions is not yet supported in customer projects. Is it still able to load the best_effort link ?

s/customer projects/chrome projects. I believe laptop projects all have SDCA Disco table support in the BIOS since it is required by Windows, too.

Reply:
@macchian, still we prefer to not add match entries if possible, so can you try the #5596 without this PR and see if you have audio probed?

Reply:
Hrm, this is for PTL, not NVL, right? Do we have the topology already built? We must have the topology built before we can add a new match entry (plus the -2ch and -4ch variants).

@bardliao, @kv2019i, should we allow more monolithic topologies to be added?